src/guides/UI_UX_Design_Guide/README.md

```diff
@@ -0,0 +1,250 @@
+
+**NOTE:** testing with at least 5 users will give you sufficient feedback on your designs (these users can be fellow students, etc.) 
+
+Resources to get your started:
+* [Article - A comprehensive guide on prototype testing](https://maze.co/blog/prototype-testing/#:~:text=Prototype%20testing%20is%20the%20process,meets%20user%20needs%20and%20expectations)
+
+</br>
+
+## Working in Figma <a name="workinginfigma"></a>
+
+**Purpose:** use Figma to create your low and high fidelity prototypes that you can share with your team. 
+
+[Figma's official website](https://help.figma.com/hc/en-us ) offers a ton of learning resources to guide you on your Figma learning journey!
+
+![alt_text](images/figmascreen.png "Figma screen")
+
+An important consideration as you work on your designs is to share your Figma file with your team members to **get feedback on the technical feasibility of your designs.** 
+
+Resources to get started with Figma: 
+* [Video - Figma for beginners](https://www.youtube.com/watch?v=FTFaQWZBqQ8)
+
+</br>
+
+Knowing how to use various things in Figma like: components, auto layout, and grids will help you design more efficiently. 
+
+</br>
+
+### Using Components
+
+“Components are elements you can reuse across your designs. They help to create and manage consistent designs across projects.” - Figma 
+
+![alt_text](images/component.png "Figma component")
+
+Here’s a great resource to get your stated: 
+* [Article + video - Components](https://help.figma.com/hc/en-us/articles/360038662654-Guide-to-components-in-Figma#:~:text=Components%20are%20elements%20you%20can,icons%2C%20layouts%2C%20and%20more)
+
+</br>
+
+### Using Auto Layout 
+
+Auto layout is important for creating responsive interfaces. “It lets you create designs that grow to fill or shrink to fit, and reflow as their contents change” - Figma
+ 
+Resources for learning auto layout: 
+* [Article - Using auto layout](https://help.figma.com/hc/en-us/articles/5731482952599-Using-auto-layout)
+* [Video - Auto layout tutorial](https://www.youtube.com/watch?v=42uQGucQA9o)
+
+</br>
+
+### Using Grids
+
+Using grids will provide visual structure to your designs. They’ll make your designs look clean & organized. 
+
+![alt_text](images/grid.png "Figma grids")
+
+Here’s a good rule of thumb to follow for designing layouts for different screen sizes: 
+* For desktops - 12 columns
+* For tablets - 8 columns 
+* For mobile devices - 4 columns
+
+Resources to learn more: 
+* [Article - Layout grids in Figma](https://www.figma.com/best-practices/everything-you-need-to-know-about-layout-grids/)
+* [Article - A comprehensive guide on layouts](https://bootcamp.uxdesign.cc/12-8-4-column-system-for-responsive-grids-df207a58ebc)
+
+</br>
+
+### Other Resources
```

Comment:
Can you add this resource too for design methods https://www.designmethods.co.uk/